monitoring/understanding-the-monitoring-stack.adoc

```diff
@@ -9,18 +9,26 @@ toc::[]
 include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]
 
 // Understanding the monitoring stack
-include::modules/monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/about-monitoring-stack-components.adoc[leveloffset=+1]
 include::modules/monitoring-default-monitoring-components.adoc[leveloffset=+2]
 include::modules/monitoring-default-monitoring-targets.adoc[leveloffset=+2]
 include::modules/monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+
+A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects].
+
 include::modules/monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
 
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
+* xref:../monitoring/managing-alerts.adoc#managing-alerts[Manage alerts]
+* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Review monitoring dashboards]
```

Comment:
This link is already present in the doc. Line 286. Is there an advantage to having it twice?

Reply:
You could have a dedicated section that describes he metrics, alerts and dashboard capabilities, plus the troubleshooting monitoring topic and provides links, then determine if you need an additional resources section in addition to that.

Reply:
Thinking about it further, perhaps the new section could also include a description and link to all of the other pages in the Monitoring book, in addition to the ones that are listed here. If the requirement is for a top level overview of the content in the book and links to relevant pages, then it would be useful for that to be a complete list. Perhaps the section could be called something like "Preparing to use the monitoring stack" or something similar.

Reply:
I agree with that @pneedle-rh , objective is to give the user an overall view of whats there in the entire book. If those links are present then we dont  need a repetition of Additional Resources.